packages/storybook/examples/core/Popover.stories.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState, useRef } from 'react';
+import { action } from '@storybook/addon-actions';
+
+import Popover, { PurePopover } from '@ichef/gypcrete/src/Popover';
+import Button from '@ichef/gypcrete/src/Button';
+import List from '@ichef/gypcrete/src/List';
+import ListRow from '@ichef/gypcrete/src/ListRow';
+
+export default {
+    title: '@ichef/gypcrete|Popover',
+    component: PurePopover,
+    subcomponents: {
+        'renderToLayer(closable(anchored(Popover))': Popover,
```

Comment:
> No props found for this component

I think that's because `renderToLayer()` HOC does not contain any prop type. We might need other ways to get those prop types out.

Reply:
Open an asana ticket: https://app.asana.com/0/347798529122928/1161326694404681